tests/shacl/test_shacl.py

```diff
@@ -1,15 +1,15 @@
-import pytest
 import os
+from pathlib import Path
 
-from lebedigital.validation import SCHEMA, read_graph_from_file, test_graph as graph_test, violates_shape, graph_from_path_violates_shapes
+from lebedigital.shacl.validation import SCHEMA, read_graph_from_file, test_graph as graph_test, violates_shape
 
-def test_graph_against_shape():
+def test_graph_against_shacl_shape():
     """
     Testing a data graph against a shapes graph and checking that the appropriate shapes fail.
     """
-
-    shapes_location = "../usecases/MinimumWorkingExample/emodul/validation_test/shape.ttl"
-    data_location = "../usecases/MinimumWorkingExample/emodul/validation_test/graph.ttl"
+    shacl_directory = os.path.dirname(Path(__file__))
+    shapes_location = Path(shacl_directory, 'youngs_modulus_shape.ttl')
```

Comment:
I'm a bit confused with the shapes in test and in lebeDigital. According to my understanding, there should not be a shape in test, but the shapes are defined in lebeDigital/shacl, only the example kg should be defined here. And what is not clear to me, are e planning on having a single shape file (for all potential tests, like we currently have in lebedigital/shacl/[CPTO_shacl_shape](https://github.com/BAMresearch/LebeDigital/blob/93-shacl-validation/lebedigital/shacl/CPTO_shacl_shape.ttl. or as in tests a specific one (e.g. for youngs_modulus)?

Reply:
Right now the test shapes are three simple checks to see if the validation code runs as expected. Which is why there are two shapes files, one to test the code, one for the doit tasks in MinimumWorkingExample. We can change the tests to actually test the shapes themselves, instead of the code. I would need a 'correct' KG then, on which i could test the shapes.

As for the amount of shapes files, I think that maybe it would make sense to have a file per 'type' of KG (i'm lacking conceptual knowledge here though)? To prevent the shapes file from becoming too large. Right now, I've just named the shape according to the ontology file that i used to create it.

Reply:
Ok, my suggestions would be to leave the code testing as is, and add the shapes according to the experimental type that we are defining later on. We will split the diagrams.net file into submodules (one for youngs modulus, one for mixture, ,..). So the CPTO_shacl_shape.ttl would have to be decomposed. Just to avoid to set the wrong structure, could we just exemplarily name it youngs_modulus_shacl_shape.ttl and once we have the KG generated test this approach)?

Reply:
Yes, I've renamed the file